pages/common/jhipster.md

```diff
@@ -0,0 +1,36 @@
+# jhipster
+
+> Generates a monolithic or microservices project after prompting user.
+> More information: <https://www.jhipster.tech/>.
+
+- Generate a simple full-stack project (monolithic or microservices):
+
+`jhipster`
+
+- Generate a simple frontend project (with OPTION = --skip-server) or a simple backend project (with OPTION = --skip-client):
+
+`jhipster [OPTION]`
+
+- Upgrade a generated project with the last JHipster updates:
+
+`jhipster upgrade`
+
+- Add a new entity to a generated project:
+
+`jhipster entity {{entity_name}}`
+
+- Import a JDL file to configure your application (entities...) - See: https://start.jhipster.tech/jdl-studio/:
```

Comment:
```suggestion
- Import a JDL file to configure your application:
```

I think it should not be a link in the command description, and the parentheses may be unnecessary.